pandas/tests/reshape/test_pivot.py

```diff
@@ -2529,6 +2529,63 @@ def test_pivot_table_aggfunc_nunique_with_different_values(self):
 
         tm.assert_frame_equal(result, expected)
 
+    def test_pivot_table_index_and_column_keys_with_nan(self, dropna: bool) -> None:
```

Comment:
Have you checked that this is not already tested?

Reply:
I looked at the other `pivot_table` tests that used `dropna` but didn't see any explicitly calling out the index/column keys as NA with all other parameters set to default. [This one](https://github.com/pandas-dev/pandas/blob/b64f438cc8079d441331396fbac1e2dc61b26af9/pandas/tests/reshape/test_pivot.py#L235) is the closest, but it sets `observed=False` instead of `observed=True` (default).